### DISCUSSION TITLE
Further development: scroll horizontally

### DISCUSSION THREAD

**Discussion by Stronkness** [ItJustWorksTM/EiffelVis#105] [Ideas]
Me and my team agreed on something that a scroll for horizontal movement on the front-end (instead of draging) would be something really appreciated. As we are not experienced in front-end development we can't solve it right now.

Would be nice to see this in the future.

**Comment by RuthgerD** [comment]
Yes we'd like some extra navigation as well, however the antv/g6 graph rendering library we use is rather unwieldy hence we kept it as simplistic as possible to avoid any friction.

But it should possible in theory :)
